apps/auth/config/index.ts

```diff
@@ -15,6 +15,8 @@ export const refreshTokenSecret = process.env.REFRESH_TOKEN_SECRET || 'a secret
 export const tokenExpiration = process.env.TOKEN_EXPIRATION || '10m';
 export const refreshTokenExpiration = process.env.REFRESH_TOKEN_EXPIRATION || '30d';
 export const cookieTokenVerificationTime = Number(process.env.COOKIE_TOKEN_VERIFICATION_TIME || TEN_MINUTES);
+export const processedCookiExpiration =Number(process.env.PROCESSED_COOKIE_EXPIRATION || 30);
```

Comment:
missing `e` on `cookie` (you wrote `Cooki`).
also, please fix the space after the equal sign.